pkg/controller/scylladbmonitoring/controller.go

```diff
@@ -261,6 +271,28 @@ func (smc *Controller) deleteScyllaDBMonitoring(obj interface{}) {
 	)
 }
 
+func (smc *Controller) addScyllaOperatorConfig(obj interface{}) {
+	smc.handlers.HandleAdd(
+		obj.(*scyllav1alpha1.ScyllaOperatorConfig),
+		smc.handlers.EnqueueOwner,
```

Comment:
It's a singleton object it doesn't have any owner, we should just enqueue all ScyllaDBMonitorings
https://github.com/scylladb/scylla-operator/blob/69375cff1653b9e40805e68372513b8e106beb40/pkg/controller/scyllaoperatorconfig/resource.go#L11-L20

Reply:
fixed, thx